2021-05-03-announcements.md

```diff
@@ -0,0 +1,35 @@
+# CFDE Announcements
+
+## CFDE Program Meeting
+A major CFDE Program meeting is scheduled for **May 10-11, 2021** with presentations and activities taking place from **8am-1pm PDT / 11am-4pm EDT** on both days.  All CFDE participants are encouraged to attend two days of workshops and talks on a range of topics. A calendar invitation has already gone out. A special notice with the agenda is forthcoming. Stay tuned!
+
+## Cross-Pollination Events
+The next event will be on **Tuesday, May 4, at 11am PDT / 2pm EDT**. Representatives from [IDG, SPARC, and the NIH](https://docs.google.com/document/d/1-PnicgLyfmisqd2pxt5yLrhcuBZeqbQqFWVO9dgvnj0/edit?usp=sharing) will be presenting. Regular Cross-Pollination events take place on the first Tuesday of every month. Please check out the [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) and let us know if you would like to present! When requesting a slot please include your preferred date and the length of your talk. Currently we have speakers scheduled out through June, but you can book a slot from July onwards. 
+
+## CFDE Trainings and Workshops
+On **May 5 at 10am PDT / 1pm EDT** we will be hosting our popular [Introduction to Amazon Web Services](https://www.nih-cfde.org/events/introduction-to-amazon-web-services-workshop-3/?pk_campaign.anc) workshop. Register by clicking on the link. Please check out the [CFDE Events Calendar](https://www.nih-cfde.org/events/?pk_campaign=anc) to find and register for future events. Our workshops are open to all, so feel free to share this information with interested friends and colleagues.
+
```

Comment:
Next week, we will be hosting a 2-day workshop on **May 12 and 14, starting at 9am PDT / noon EDT both days**, to introduce workflows using Snakemake. Please register for [day 1](https://www.nih-cfde.org/events/introduction-to-workflows-using-snakemake-2-2021-05-12/?pk_campaign.anc) and [day 2](https://www.nih-cfde.org/events/introduction-to-workflows-using-snakemake-2-2021-05-14/?pk_campaign.anc)